package.json

```diff
@@ -36,7 +36,7 @@
         "typescript": "^4.8.4"
     },
     "dependencies": {
-        "@dhis2/analytics": "^26.0.7",
+        "@dhis2/analytics": "^26.0.20",
```

Comment:
Note that this bump brings with it side-effects that we might not want. So you might have to hold the merge of this until those are resolved properly (and a new Analytics version is released)

Reply:
Especially related to https://github.com/dhis2/analytics/pull/1531 and the issues found today 

1. https://dhis2.slack.com/archives/G01PW9YGZD2/p1696491389404769
2. https://dhis2.slack.com/archives/G01PW9YGZD2/p1696493354852079